service_facts: Use LANG=C to run commands

tyll opened the pull request:
##### SUMMARY
This allows to parse the output when the user's locale changes the
commands' output. For example chkconfig uses 'Ein' and 'Aus' instead of
'on' and 'off' when using LANG=de_DE.UTF-8 breaking the service
detection on RHEL 6.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
service_facts

##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
```
ansible 2.7.0.dev0 (service_facts_LANG_C be6a1236ad) last updated 2018/08/21 18:47:36 (GMT +200)
  config file = /home/till/.ansible.cfg
  configured module search path = [u'/home/till/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/till/scm/gh-ansible-ansible/lib/ansible
  executable location = /home/till/scm/gh-ansible-ansible/bin/ansible
  python version = 2.7.15 (default, May 16 2018, 17:50:09) [GCC 8.1.1 20180502 (Red Hat 8.1.1-1)]

ansible 2.6.1
  config file = /home/till/.ansible.cfg
  configured module search path = [u'/home/till/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15 (default, May 16 2018, 17:50:09) [GCC 8.1.1 20180502 (Red Hat 8.1.1-1)]

ansible 2.6.2
  config file = /home/till/.ansible.cfg
  configured module search path = [u'/home/till/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15 (default, May 16 2018, 17:50:09) [GCC 8.1.1 20180502 (Red Hat 8.1.1-1)]

```


##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful. -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```
# before
ansible -m service_facts -i rhel6-cloud, all
rhel6-cloud | SKIPPED

# after
ansible -i rhel6-cloud,  -m service_facts all
rhel6-cloud | SUCCESS => {
    "ansible_facts": {
        "services": {
            "acpid": {
                "name": "acpid", 
                "source": "sysv", 
                "state": "running"
            }, 
[...]
            "udev-post": {
                "name": "udev-post", 
                "source": "sysv", 
                "state": "running"
            }
        }
    }, 
    "changed": false
}

```

tyll commented:
ready_for_review

bcoca commented on lib/ansible/modules/system/service_facts.py:
```diff
@@ -89,7 +91,8 @@ def gather_services(self):
         if service_path is not None and chkconfig_path is None:
-            rc, stdout, stderr = self.module.run_command("%s --status-all 2>&1 | grep -E \"\\[ (\\+|\\-) \\]\"" % service_path, use_unsafe_shell=True)
+            rc, stdout, stderr = self.module.run_command(
+                "%s --status-all 2>&1 | grep -E \"\\[ (\\+|\\-) \\]\"" % service_path, use_unsafe_shell=True, environ_update=LANG_C)
```
instead of updating each command just set `run_command_environ_update`

tyll replied:
Thx, I changed this and also added LC_ALL to the override. When I understand `man 7 locale` correctly, only setting `LC_ALL` should suffice since it has the highest priority.

bcoca replied:
man page says that .. but not all programs obey that, though i think all the ones related to services do.

tyll committed:
service_facts: Use LC_ALL/LANG=C to run commands

This allows to parse the output when the user's locale changes the
commands' output. For example chkconfig uses 'Ein' and 'Aus' instead of
'on' and 'off' when using LANG=de_DE.UTF-8 breaking the service
detection on RHEL 6.

tyll commented:
ready_for_review
